kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/source/TempFileHelper.scala

```diff
@@ -0,0 +1,23 @@
+package io.lenses.streamreactor.connect.aws.s3.source
+
+import java.io.File
+import java.util.UUID
+
+trait TempFileHelper {
```

Comment:
Would be nice if this used `java.nio.file.Files.createTempDirectory` or  `createTempFile` and then call `deleteOnExit()`
as this will automatically do the cleanup when the jvm is terminated (in case, for whatever reason, the cleanup is not successful).

I'm often noticing files accumulated generated during tests.

Reply:
createTempDirectory does not exist, and my understanding is that this creates files and not directories. 
The code was guarding for deletion, but I added the .deleteOnExit .